### DISCUSSION TITLE
Documentation on jax.lax.precision

### DISCUSSION THREAD

**Discussion by PhilipVinc** [jax-ml/jax#8443] [Q&A]
It is not entirely clear to me what the `precision` argument to some `lax` operation means.
The documentation only tells me that it should be a `jax.lax.precision` object which can have one of three values (`HIGH`, `HIGHEST`, `DEFAULT`), but It is unclear to me what those mean and how they interact with input array dtypes...

Is there some extra documentation to it?

**Comment by jakevdp** [comment]
Thanks - I'm going to convert this to an issue to track updating this documentation: #8447
